### ISSUE TITLE
d3 arc and arc

### ISSUE THREAD

**Issue by domoritz** [rollup/rollup#1614]
In my code, I am using

```
import * as d3 from "d3";

var arc = d3.arc()
      .innerRadius(size + offset)
      .outerRadius(size + offset + 1)
      .startAngle(-Math.PI/6)
      .endAngle(Math.PI/6);
```

This code compiles to 

```
var arc = arc()
      .innerRadius(size + offset)
      .outerRadius(size + offset + 1)
      .startAngle(-Math.PI/6)
      .endAngle(Math.PI/6);
```

This is not correct as the variable arc gets overridden.

**Comment by kzc** [comment]
Cannot reproduce.
```
$ node_modules/.bin/rollup -v
rollup version 0.49.2
```
```
$ cat main.js
import * as d3 from "d3";

var arc = d3.arc()
      .innerRadius(size + offset)
      .outerRadius(size + offset + 1)
      .startAngle(-Math.PI/6)
      .endAngle(Math.PI/6);
```
```
$ cat rollup.config.js 
import resolve from 'rollup-plugin-node-resolve';
export default [{
    input: 'main.js',
    plugins: [ resolve(), ],
    output: [{ file: 'bundle.js', format: 'es' }],
}];
```
```
$ node_modules/.bin/rollup -c rollup.config.js

main.js → bundle.js...
created bundle.js in 1.8s
```
```
$ tail -6 bundle.js 

var arc = arc$1()
      .innerRadius(size + offset)
      .outerRadius(size + offset + 1)
      .startAngle(-Math.PI/6)
      .endAngle(Math.PI/6);
```

**Comment by guybedford** [comment]
This seems to be fixed in 0.55.4 - https://rollupjs.org/repl?version=0.55.4&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMColMjBhcyUyMGQzJTIwZnJvbSUyMCU1QyUyMmQzJTVDJTIyJTNCJTVDbiU1Q252YXIlMjBhcmMlMjAlM0QlMjBkMy5hcmMoKSU1Q24lMjAlMjAlMjAlMjAlMjAlMjAuaW5uZXJSYWRpdXMoc2l6ZSUyMCUyQiUyMG9mZnNldCklNUNuJTIwJTIwJTIwJTIwJTIwJTIwLm91dGVyUmFkaXVzKHNpemUlMjAlMkIlMjBvZmZzZXQlMjAlMkIlMjAxKSU1Q24lMjAlMjAlMjAlMjAlMjAlMjAuc3RhcnRBbmdsZSgtTWF0aC5QSSUyRjYpJTVDbiUyMCUyMCUyMCUyMCUyMCUyMC5lbmRBbmdsZShNYXRoLlBJJTJGNiklM0IlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIyZXMlMjIlMkMlMjJuYW1lJTIyJTNBJTIyJTIyJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTdEJTJDJTIyZXhhbXBsZSUyMiUzQW51bGwlN0Q=
